post-processor/amazon-import/post-processor.go

```diff
@@ -0,0 +1,277 @@
+package amazonimport
+
+import (
+	"fmt"
+	"strings"
+	"os"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/aws/aws-sdk-go/service/s3"
+// This is bad, it should be pulled out into a common folder across
+// both builders and post-processors
+	awscommon "github.com/mitchellh/packer/builder/amazon/common"
+	"github.com/mitchellh/packer/common"
+	"github.com/mitchellh/packer/helper/config"
+	"github.com/mitchellh/packer/packer"
+	"github.com/mitchellh/packer/template/interpolate"
+)
+
+const BuilderId = "packer.post-processor.amazon-import"
+
+// Configuration of this post processor
+type Config struct {
+	common.PackerConfig `mapstructure:",squash"`
+	awscommon.AccessConfig `mapstructure:",squash"`
+
+// Variables specific to this post processor
+	S3Bucket	string `mapstructure:"s3_bucket_name"`
+	S3Key		string `mapstructure:"s3_key_name"`
+	SkipClean	bool   `mapstructure:"skip_clean"`
+	ImportTaskDesc	string `mapstructure:"import_task_desc"`
+	ImportDiskDesc	string `mapstructure:"import_disk_desc"`
+	Tags		map[string]string `mapstructure:"tags"`
+
+	ctx interpolate.Context
+}
+
+type PostProcessor struct {
+	config Config
+}
+
+// Entry point for configuration parisng when we've defined
+func (p *PostProcessor) Configure(raws ...interface{}) error {
+	p.config.ctx.Funcs = awscommon.TemplateFuncs
+	err := config.Decode(&p.config, &config.DecodeOpts{
+		Interpolate:		true,
+		InterpolateContext:	&p.config.ctx,
+		InterpolateFilter:	&interpolate.RenderFilter{
+			Exclude: []string{},
+		},
+	}, raws...)
+	if err != nil {
+		return err
+	}
+
+	// Set defaults
+	if p.config.ImportTaskDesc == "" {
+		p.config.ImportTaskDesc = fmt.Sprintf("packer-import-%d", interpolate.InitTime.Unix())
```

Comment:
Ah, I had borrowed that logic from the virtualbox builder here:
https://github.com/mitchellh/packer/blob/master/builder/virtualbox/ovf/config.go#L72 without realizing that InitTime might be for the post-processor rather than the start of the build.

I will update the pull with the method in the compress post-processor.